.github/scripts/commit/assert_commit_message.py

```diff
@@ -18,6 +18,9 @@ def assert_commit_message(branch_name):
     (False, "100 Configure"),
     (False, " #100 Configure"),
     (False, "#100  Configure"),
+    (True, "Configure"),
+    (True, "Configure kotest long message"),
+    (False, "Configure kotest "),
```

Comment:
Really nice that you've handled empty spaces as well 🍻

Reply:
🍻